src/site_upload/study_period/study_period.py

```diff
@@ -11,12 +11,23 @@
 
 def update_study_period(s3_client, s3_bucket, site, study, data_package, version):
     """gets earliest/latest date from study metadata files"""
-    path = awswrangler_functions.get_s3_study_meta_list(
+    paths = awswrangler_functions.get_s3_study_meta_list(
         s3_bucket, study, data_package, site, version
     )
-    if len(path) != 1:
-        raise KeyError("Unique date path not found")
-    df = awswrangler.s3.read_parquet(path[0])
+    if len(paths) > 1:
+        latest_key = None
+        latest_date = datetime(1000, 1, 1, tzinfo=UTC)
```

Comment:
There is `datetime.min` (`== datetime.datetime(1, 1, 1, 0, 0)`) which might the intent here more obvious.